web/src/app/profile/ProfileContainer.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import ProfileComponent from './ProfileComponent';
+import PropTypes from 'prop-types';
+import { Redirect } from 'react-router-dom';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { profileOperations } from './duck';
+
+/* global sessionStorage */
+class ProfileContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      logout: false
+    };
+
+    this.logout = this.logout.bind(this);
+  }
+  componentWillMount() {
+    const { getUser } = this.props;
+    if (!sessionStorage.getItem('user')) {
+      getUser();
+    }
+  }
+
+  logout() {
+    // TODO: call backend to logout user
```

Comment:
This is done by submiting a post request to springs /logout endpoint... I think we may need to have CSRF tokens enabled for this to work though. I can handle the logout stuff. Will make a trello task for it

Reply:
For sure for sure